update_list.py

```diff
@@ -0,0 +1,351 @@
+import argparse
+import json
+import re
+from enum import Enum
+from itertools import cycle
+from pathlib import Path
+from typing import Union
+
+import requests
+from user_agents.parsers import UserAgent, parse
+
+
+class Browser(Enum):
+    CHROME = "chrome"
+    FIREFOX = "firefox"
+    EDGE = "edge"
+    SAFARI = "safari"
```

Comment:
I would make this a `str, Enum`! The advantage is that members behave like `str`s (their value) and you don't need `.value` everywhere. It also allows lazy users to pass the strings where you type-hint `Browser`, but 🤫, don't tell them that.

Edit: BTW, if we're going to have some predefined browsers, maybe we can re-add the type-hint I wanted to add in #346.